python/foxglove-sdk/python/foxglove/channel.py

```diff
@@ -87,24 +91,55 @@ def close(self) -> None:
 _channels_by_topic: Dict[str, Channel] = {}
 
 
-def log(topic: str, message: Any) -> None:
-    channel: Optional[Channel] = _channels_by_topic.get(topic, None)
+def log(
+    topic: str,
+    message: Union[JsonMessage, list[Any], bytes, str, schemas.FoxgloveSchema],
+    *,
+    log_time: Optional[int] = None,
+    publish_time: Optional[int] = None,
+    sequence: Optional[int] = None,
+) -> None:
+    """Log a message on a topic.
+
+    Creates a new channel the first time called for a given topic.
+    For foxglove types in the schemas module, this creates a typed channel.
```

Comment:
capitalize "Foxglove"

Reply:
could link to the typed channels module to give "typed channel" some context. something like:


```
(see :py:mod:`foxglove.channels`)
```